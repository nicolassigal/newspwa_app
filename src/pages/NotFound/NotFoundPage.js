import React from 'react';

const NotFoundPage = () => {
    return <div className="not-found-page">Not Found</div>
}

export default NotFoundPage;