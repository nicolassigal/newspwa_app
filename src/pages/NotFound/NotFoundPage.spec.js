import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import NotFoundPage from './NotFoundPage';

describe('Not Found Page', () => {
  it('Not Found page should render correctly', () => {
    const tree = renderer
      .create(<NotFoundPage />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});