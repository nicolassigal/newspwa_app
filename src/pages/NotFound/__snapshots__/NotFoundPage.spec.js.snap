// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not Found Page Not Found page should render correctly 1`] = `
<div
  className="not-found-page"
>
  Not Found
</div>
`;
